marklogic-junit5/examples/simple-ml-gradle/build.gradle

```diff
@@ -15,11 +15,11 @@ repositories {
 }
 
 dependencies {
-  mlBundle "com.marklogic:marklogic-unit-test-modules:1.0.0"
+  mlBundle "com.marklogic:marklogic-unit-test-modules:1.1.0"
 
-	compile "com.marklogic:marklogic-client-api:4.2.0"
+	api "com.marklogic:marklogic-client-api:5.4.0"
 
-	testCompile "com.marklogic:marklogic-junit:1.0.0"
+	testImplementation "com.marklogic:marklogic-junit:1.0.0"
```

Comment:
Shouldn't this be 1.1.0?  Or are we not planning to increment the release number here?

Reply:
Not yet, when we merge to master we can bump it according to Rob

Reply:
What I like to do is once a release is ready and able to be merged to master, I make one final commit to develop that updates the version number to the new release number. Then I merge all of that onto master (can often fast-forward master to develop too).

Reply:
That's fine, but why did we bump the marklogic-junit5 version in the same sample file a few lines above?

This isn't an active build file, it's just an example, effectively for documentation.  It seems like we'd want to be consistent at each step in the process.  Maybe marklogic-junit5 dependency above should use the snapshot for consistency until we update everything for the full release version?

Reply:
Would you be more specific as I am getting confused. By the way when I google  marklogic-junit:1.0.0 does not exist in maven repository. What I suggest is when we merge to master we can take care of these differences.

Reply:
Sorry, I was confused.  I'm not sure if we're planning to make any future changes to marklogic-junit module any more.  I'll go ahead and merge this in for now.